webpack/scenes/Subscriptions/Manifest/__tests__/CdnConfiguration.test.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { cleanup } from '@testing-library/react';
+import { renderWithRedux, fireEvent, patientlyWaitFor } from 'react-testing-lib-wrapper';
+import CdnConfiguration from '../CdnConfiguration';
+import { nockInstance, assertNockRequest } from '../../../../test-utils/nockWrapper';
+import { updateCdnConfigurationSuccessResponse } from '../../../Organizations/__tests__/organizations.fixtures';
+import api from '../../../../services/api';
+
+afterEach(cleanup);
+const updateCdnConfigurationPath = api.getApiUrl('/organizations/1/cdn_configuration');
+
+const cdnConfiguration = {
+  ssl_ca_credential_id: 2,
+};
+
+const defaultProps = {
+  cdnConfiguration: {
+
+  },
+};
```

Comment:
Shouldn't this match the required PropTypes shape?
```js
cdnConfiguration: PropTypes.shape({
    url: PropTypes.string,
    username: PropTypes.string,
    upstream_organization_label: PropTypes.string,
    ssl_ca_credential_id: PropTypes.number,
    password_exists: PropTypes.bool,
  }).isRequired,
```
Or if it doesn't matter, at least clean up a bit ;)
```suggestion
const defaultProps = {
  cdnConfiguration: {},
};
```

Reply:
Not according to my tests - but I can add one